ocaml/testsuite/tests/typing-layouts/basics.ml

```diff
@@ -2255,9 +2255,9 @@ let poly : ('a. 'a -> 'a) -> int * bool =
   fun (id : ('a : immediate). 'a -> 'a) -> id 3, id true
 
 [%%expect{|
-Line 2, characters 7-38:
+Line 2, characters 6-39:
 2 |   fun (id : ('a : immediate). 'a -> 'a) -> id 3, id true
-           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This change is ok, but do you know what caused it and if it can be avoided? (No need to spend too much time, tho)